Eos/Soave-Redlich-Kwong/EOS.H

```diff
@@ -48,14 +41,12 @@ MixingRuleAmBm(amrex::Real T, amrex::Real Y[], amrex::Real &am, amrex::Real &bm)
 {
   am = 0.0;
   bm = 0.0;
-  amrex::Real Tr ;
+  amrex::Real sqrtT = std::sqrt(T);
```

Comment:
Do you want to "protest" this with an assert or something here and below?

Reply:
Yeah, if you have a negative temperature you're going to have a bad time, checking for that couldn't hurt. I put in some AMREX_ASSERTs to capture this.